recipe/meta.yaml

```diff
@@ -37,6 +37,10 @@ test:
 
 outputs:
   - name: cuda-cudart
+    build:                                               # [win]
+      missing_dso_whitelist:                             # [win]
+        - "*/api-ms-win-core-libraryloader-*.dll"        # [win]
+        - "*/api-ms-win-security-systemfunctions-*.dll"  # [win]
```

Comment:
For now have listed these so packages get built and uploaded. Though ideally we wouldn't need to list these packages and they would have been found through the compiler runtime libraries on Windows. If we can figure out and resolve these missing libraries, we can remove this section

Reply:
I would think this is acceptable.

Reply:
FWIW there is [`api-ms-win-core-libraryloader-l1-1-0.dll` in the repack]( https://github.com/conda-forge/vc-feedstock/blob/75f9129f48691789cb08065dc4698190fe9b539e/recipe/meta.yaml#L89 ), but the build here is looking for `api-ms-win-core-libraryloader-l1-2-0.dll`

There is no `api-ms-win-security-systemfunctions-*.dll` in the repack

Reply:
After discussion below ending with this comment ( https://github.com/conda-forge/cuda-cudart-feedstock/pull/2#issuecomment-1751781183 ), it appears this is reasonable in the end. Marking this resolved